main.go

```diff
@@ -218,7 +221,13 @@ type AtomEntry struct {
 
 func downloadFeed(url string) ([]byte, error) {
 	log.Printf("downloading feed %#v\n", url)
-	resp, err := http.Get(url)
+	client := http.Client{}
```

Comment:
not sure what the vim incantation is, but i'd move this closer to it's first use via `client.Do` -- say `m5` or something should do it in vim? 😜

Reply:
Good idea. Done!

(I used a hex editor, vim is too high level for me)

Reply:
I assume that's hexl-mode? 😝

Reply:
But I think i'm missing the actual move?

Reply:
:eagle: :eye:  ... `commit` without `add` will do that :facepalm:

Reply:
😉  cheers!